Helm/opensearch/templates/statefulset.yaml

```diff
@@ -257,14 +292,26 @@ spec:
           - name: node.{{ $role }}
             value: "{{ $enabled }}"
           {{- end }}
+{{- if .Values.extraEnvs }}
+{{ toYaml .Values.extraEnvs | indent 10 }}
+{{- end }}
+{{- if .Values.envFrom }}
+        envFrom:
+{{ toYaml .Values.envFrom | indent 10 }}
+{{- end }}
+        volumeMounts:
+          {{- if .Values.persistence.enabled }}
+          - name: "{{ template "opensearch.uname" . }}"
+            mountPath: /usr/share/opensearch/data
```

Comment:
Can we parameterize the path instead of hardcoding it?

Reply:
@belfhi

Reply:
@belfhi would you be able to make this change and update the PR?